docs/ref-info/ref-integration-approaches.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Integration Approaches
+description: Reference information about the integration approaches available for UID2 integration.
+hide_table_of_contents: false
+sidebar_position: 06
+---
+
+import Link from '@docusaurus/Link';
+
+# Integration Approaches
+
+For publishers integrating UID2 into their workflows, or advertisers integrating UID2 support, there are three broad integration approaches. UID2 integrations can be implemented entirely on the client side, entirely on the server side, or partially on the client side and partially on the server side (client-server).
+
+For details, see:
+
+- [Client-side integration](#client-side-integration)
+- [Client-server integration](#client-server-integration)
+- [Server-side integration](#server-side-integration)
+
+## Client-Side Integration
+
+In a client-side integration, UID2 tokens are generated and refreshed on the client side.
+
+For example:
+
+- Publishers generate UID2 tokens on the client side for bidstream use, as well as refreshing the tokens.
+- Advertisers generate UID2 tokens on the client side for tracking pixels.
+
+Examples of documentation for publisher client-side integrations:
+
+- [UID2 Client-Side Integration Guide for Prebid.js](../guides/integration-prebid-client-side.md)
+- [Client-Side Integration Guide for JavaScript](../guides/integration-javascript-client-side.md)
```

Comment:
off-topic - should this be renamed to **UID2** Client-Side Integration Guide for Javascript to keep it consistent? I think originally i suggested to remove the term UID2 but now it seems to be the odd one out...

Reply:
@sunnywu Not sure which way to go on this. Noting this inconsistency for further consideration and a near future fix on a separate Jira.